tests/cpp/test_matmul_scheduler.cpp

```diff
@@ -2699,6 +2743,66 @@ TEST_F(MatmulSchedulerTest, DISABLED_RequireExternalPlugin) {
   MatmulParams params;
 }
 
+// Test that we can segment a Fusion containing two matmuls
+TEST_F(MatmulSchedulerTest, Llama2FFN) {
+  NVFUSER_TEST_CUDA_ARCH_RANGE_GUARD(7, 5, 9, 0);
+
+  for (bool enable_fusion : {false, true}) {
```

Comment:
I would recommend parametrizing this, the parametrization pattern can be used with the `SegmentMatmulOpPrologue` and `SegmentLinearOpPrologue` tests as well to test with all three combinations of schedulers as listed in this comment --
```

// TODO: Once we can control the ExprEval and Matmul schedulers via options, run
// this test with all three combinations (with and without each scheduler, but
// at least one enabled).

Reply:
Oh that's a good idea

Reply:
I added a parametrization for the Llamma2FFN test, but the prologue tests are actually currently failing when fusion is enabled, so I will leave that for a separate PR (that might be fixed by #2309).